docs/content/about/_index.md

```diff
@@ -15,6 +15,14 @@ aliases:
   - /about/goals/
 ---
 
+## Our Vision
+
+NIST and the community will maintain a set of OSCAL models for organizations and service providers to facilitate seamless, continuous security assessment by exchanging information about system implementation and analysis without vendor lock-in. ​
+
+Community engagement and collaboration will mature OSCAL models faster and drive the broadest international adoption possible in public and private sectors.
+
+## What is OSCAL?
+
 NIST is developing the Open Security Controls Assessment Language (OSCAL) as a standardized, data-centric framework that can be applied to an information system for documenting and assessing its security controls. Today, security controls and control baselines are represented in proprietary formats, requiring data conversion and manual effort to describe their implementation. An important goal of OSCAL is to move the security controls and control baselines from a text-based and manual approach (using word processors or spreadsheets) to a set of standardized and machine-readable formats. With systems security information represented in OSCAL, security professionals will be able to automate security assessment, auditing, and continuous monitoring processes.
```

Comment:
NIST replaced 'information systems' with 'systems' in the RMF-related SPs. Suggest aligning here too.
Suggestion: The text: "to a set of standardized and machine-readable formats." should read "to a set of standardized machine-readable formats."
Text: "With systems security information represented in OSCAL, ..." should read "With systems' security information .....

Reply:
Sending up a PR with these recommendations integrated.